R/differential_colocalization_analysis.R

```diff
@@ -68,14 +84,21 @@ RunDCA.data.frame <- function (
         )
     }
   }
+
+  stopifnot(
+    "'cl' must be a cluster object or an integer" =
+      inherits(cl, what = c("cluster", "numeric"))
```

Comment:
The default value is NULL now, which triggers this!

Reply:
That should be fixed now!